userdocs/src/usage/applicationyaml.md

```diff
@@ -1,22 +1,44 @@
 
 ## Motivation
 
-Something catchy
+Deploying an application in Kubernetes can be challenging, especially if one is new to the declarative paradigm. More so 
+when dealing with the concept of infrastructure as code through a technology like ArgoCD for the first time. Add a new 
+kind of Ingress controller combined with the AWS platform for provisioning certificates, and you have yourself a jolly
+good time in research land. 
+
+Our deployment process aims to fix that. 
+
+1. First you create and configure an application declaration. This declaration can be deleted after applying it. You'll
+be making changes to the Kubernetes and ArgoCD resources from that point on.
+2. We apply the application declaration to the infrastructure as code repository, which converts the application
+declaration to resources understood by Kubernetes and ArgoCD.
 
 ## Commands
 
 To create an application.yaml template, run the following command:
 
-`okctl create application > application.yaml`
+`okctl create application <env>`
```

Comment:
The syntax we have used in okctl command specyfies arguments as upper case. So here it would be `okctl create appliaction ENV`. This is an MD file and not the CLI, but I think consistency makes sense here as well.